source/agora/node/Ledger.d

```diff
@@ -765,6 +762,56 @@ public class Ledger
         }
 
         // Finally, validate the signatures
+        return this.validateBlockSignature(block);
+    }
+
+    /***************************************************************************
+
+        Validate the signature of a block
+
+        This validate that the signature in a block header is consistent with
+        the enrolled validators, and cryptographically correct.
+        Note that since this requires to know which nodes are validators,
+        this method is contextful and can only guarantee the signature
+        of the next block, as the validator set might change after that.
+
+        Implementation_details:
```

Comment:
The underscore is not necessary.

Reply:
No it's not, it's how DDOC recognize a section name. When generating the documentation, a second name starts with a single "word". Any underscores are stripped.